Save board issues

anthonwellsjo opened the pull request:
Har gjort en mycket mer lättanvänd och orörig implementering av save board -funktionen:

tagit bort spara-knapp:

- varje kort sparas enskilt när man trycker spara på kortet 
- inga problem med dubletter om man öppnar och stänger flera gånger

issues:

- kvarliggande issue från förut - implementera ta-bort-funktion så att den också uppdaterar localstorage




close #55 
close #44 
close #54 



tidigare funktioner ligger kvar bortkommenterade .... tas bort när issues är lösta

anthonwellsjo committed:
solved issue #54

anthonwellsjo committed:
simpler save card functionality

anthonwellsjo committed:
snabb kommit åka buss

anthonwellsjo committed:
ny implementering av sparning fungerande

anthonwellsjo committed:
formatting

firefool reviewed (dismissed):
Verkar funka bra nu! Korten hamnar i bakvänd ordning när dom laddas från localstorage men det borde ju vara lätt att fixa?

anthonwellsjo committed:
commit to meerge with head

anthonwellsjo committed:
Merge branch 'dev' of https://github.com/anthonwellsjo/Kanban-Meta-Grupp-3 into saveBoardIssues

anthonwellsjo committed:
commit new save function works

anthonwellsjo committed:
unikt card-id attribut i page.addCard

varje kort sparas som ett objekt med namnet = card-id. därför sparas gammalt kort över av det nya

anthonwellsjo commented:
Tjena,


blockade min PR för jag hittade några fula buggar som hade med e.target att göra ===> olika utgångspunkt om man tryckte på nytt kort eller om man redigerade en existerande ===>> vilket ledde till att jag man hämtade ut fel namn på kolumnen och därmed sparades kortet i fel kolumn.

har sedan fixat ett mycket bättre sätt för att inte skapa dubbletter... varje kort-html får en attribut i sitt html-dokuement: card-id som sedan används för att skriva över gammalt kort.

efter denna merge fortsätter jag sparning av ordningen.

anthonwellsjo committed:
löste konflikter

anthonwellsjo commented:
okej se om det funkar för er... här funkar det

firefool reviewed (approved):
Sparandet verkar funka fint nu!
